ui/app/adapters/job.js

```diff
@@ -86,11 +86,16 @@ export default class JobAdapter extends WatchableNamespaceIDs {
     });
   }
 
-  update(job) {
+  update(job, format) {
```

Comment:
Noted this in the [umbrella ticket](https://github.com/hashicorp/nomad/issues/15982#issuecomment-1490802958). It might be worth going through the larger body of work prior to merge and then handling all areas where JSDoc is appropriate together.